"""custom errors in this package"""

from prescyent.utils.errors.custom_exception import CustomException
from prescyent.utils.errors.dataset import DatasetEmptyException
from prescyent.utils.errors.predictor import PredictorNotFound, PredictorUnprocessable
